packages/block/CHANGELOG.md

```diff
@@ -200,11 +205,14 @@ You can instantiate a Merge/PoS block like this:
 
 ```typescript
 import { Block } from '@ethereumjs/block'
-import Common, { Chain, Hardfork } from '@ethereumjs/common'
-const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Merge, })
-const block = Block.fromBlockData({
-  // Provide your block data here or use default values
-}, { common })
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Merge })
+const block = Block.fromBlockData(
+  {
+    // Provide your block data here or use default values
+  },
+  { common }
+)
```

Comment:
Yeah I left the changes in the .md files in a separate commit so that they're easily removable, but I'm unsure why a lot of this sort of formatting was automatically done, I assumed it was project-configured formatting that only got triggered now that changes were made to these files? Anyhow will revert most of these, if we want to improve .md formatting we can always do it in another PR.

Reply:
👍